custom_components/google_home/const.py

```diff
@@ -89,8 +90,128 @@
 # JSON parameter values when retrieving information from devices
 JSON_ALARM: Final[str] = "alarm"
 JSON_TIMER: Final[str] = "timer"
-JSON_NOTIFICATIONS_ENABLED: Final[str] = "notifications_enabled"
 JSON_ALARM_VOLUME: Final[str] = "volume"
+JSON_NOTIFICATIONS_ENABLED: Final[str] = "notifications_enabled"
+JSON_EUREKA_DEVICE_INFO: Final[str] = "device_info"
+JSON_EUREKA_CAPABILITIES: Final[str] = "capabilities"
+
+# EUREKA
+EUREKA_KEY_PARAMS = "params"
+EUREKA_KEY_OPTIONS = "options"
+EUREKA_PARAMS = (
+    "version,audio,name,build_info,detail,device_info,net,wifi,setup,settings,"
+    "opt_in,opencast,multizone,proxy,night_mode_params,user_eq,room_equalizer,"
+    "sign,aogh,ultrasound,mesh "
+)
+EUREKA_OPTIONS = "detail"
+EUREKA_JSON_DEVICE_INFO_4K_BLOCKED = "4k_blocked"
```

Comment:
I think it's cleaner to define non final TypedDict for returned JSON instead all of these. It can be also used instead of `EurekaCompatibilityDict`. Should allow to remove significant part of the code.